src/php/install.sh

```diff
@@ -156,6 +197,33 @@ install_php() {
     mkdir -p $PHP_SRC_DIR
     cd $PHP_SRC_DIR
     wget -O php.tar.xz "$PHP_URL"
+}
+
+install_php() {
+    PHP_VERSION="$1"
+    PHP_INSTALL_DIR="${PHP_DIR}/${PHP_VERSION}"
+    if [ -d "${PHP_INSTALL_DIR}" ]; then
+        echo "(!) PHP version ${PHP_VERSION} already exists."
+        exit 1
+    fi
+
+    if ! cat /etc/group | grep -e "^php:" > /dev/null 2>&1; then
+        groupadd -r php
+    fi
+    usermod -a -G php "${USERNAME}"
+    PHP_URL="https://www.php.net/distributions/php-${PHP_VERSION}.tar.gz"
+
+    PHP_INI_DIR="${PHP_INSTALL_DIR}/ini"
+    CONF_DIR="${PHP_INI_DIR}/conf.d"
+    mkdir -p "${CONF_DIR}";
+
+    PHP_EXT_DIR="${PHP_INSTALL_DIR}/extensions"
+    mkdir -p "${PHP_EXT_DIR}"
+
+    PHP_SRC_DIR="/usr/src/php"
+    mkdir -p $PHP_SRC_DIR
+    cd $PHP_SRC_DIR
+    wget -O php.tar.xz "$PHP_URL" || install_previous_version
```

Comment:
We use `install_php` to install `version` as well as `additionalVersions`. We don't check that the fallback logic is applied only if the `version` is `latest`. This will lead into discrepancies as we talked before. Can we update the logic?

Reply:
done